librz/util/bitmap.c

```diff
@@ -34,9 +42,11 @@ RZ_API RZ_OWN RzBitmap *rz_bitmap_new(size_t len) {
 
 RZ_API void rz_bitmap_set_bytes(RZ_NONNULL RzBitmap *b, RZ_NONNULL const ut8 *buf, size_t len) {
 	rz_return_if_fail(b && buf && len >= 0);
-	if (b->length < len) {
-		len = b->length;
+	size_t blen = b->length << BITWORD_BITS_SHIFT;
```

Comment:
isn't it better to use BITMAP_WORD_COUNT on `len` and compare that?

Reply:
is the same.